### DISCUSSION TITLE
Seattle Development and Home Sale Prices

### DISCUSSION THREAD

**Discussion by kevography** [jakobzhao/geog495#45] [Final Project]
**Application:** https://kevography.github.io/seattle-development-map/
**Repository:** https://github.com/kevography/seattle-development-map
**Description:** A map visualizing residential units that completed construction in a specified year and the average price per square foot of single-family homes sold in that year by census tract. By using the slider users can select years in the range of 2006-2015 as that is the range of years represented in **both** sets. Zooming in allows the users to see the polygon representations of each development site. Viewing through specific years allows users to see trends in development patterns and year-to-year sale prices of single-family homes.
![devmap-img1](https://user-images.githubusercontent.com/77176977/145138163-f2aa6760-42a4-4fd4-a910-f1ceb46fe2b2.jpg)

**Comment by VaneRaklan** [comment]
I think that the custom base map looks really good and provides an excellent contrast to the visualizations that you are making. On top of that I think your side panel looks very clean for how much is there.
Two things that I was thinking were that the color scale doesn't look like it shows enough difference between census tracts. I'm not sure if a fix would be exaggerating the numbers (i.e. squaring them) just for the purpose of the scale's domain or if using a diverging color scale would do the trick.
Also, it seems like there's a lot of land uses from the legend that either aren't present or aren't showing up in the map so that might be something to address.

**Comment by jacobf19** [comment]
I really like this map. It shocks me to see how expensive land is in Seattle. I do have a couple suggestions. I would try to figure out how you can bring the labels of the basemap above the data layer. The layer is currently blocking the labels, while others are visible around the city boundaries. It may also be helpful to describe why there is no data in the University District or in Downtown Seattle.

**Comment by manhongg** [comment]
Your project idea was very ambitious and a unique approach to web mapping that I haven't really seen in this class. Your map was quite informative and eye-opening in terms of conveying the data visually. Perhaps due to the level of detail in your web app, I felt like the sidebar was a bit busy and could be simplified to be more intuitive and user-friendly. Overall, I really liked your approach to this project and the level of detail that went into the web app! Great job!

**Comment by Zhiyuan-Jiao** [comment]
Hello, this is really a cool map with a lot of information presented to the audience. One thing that ii believe could be improved is that maybe you could try to make all the land use type into a filter function so that the side bar won't be so busy and the user could still get to view the different land uses.

**Comment by RensenJiang** [comment]
A great and informative map to combine multiple factors into one map. I think you can make the map better by adding a filter function to factors shown on the map, so users could help a better view when they are only looking for specific factors.
